src/js/components/stock-movement-wizard/AddItemsPage.jsx

```diff
@@ -1100,7 +1100,7 @@ class AddItemsPage extends Component {
   }
 
   render() {
-    const showOnly = this.props.showOnly && this.state.values.statusCode !== 'CREATED';
+    const showOnly = this.props.showOnly && (this.state.values.statusCode !== 'CREATED' || !request);
```

Comment:
@DominikaHatala this is old file kepnt only fo rebase pursposes. This file is not used right now.

Reply:
Ohhh nvm, this is to develop XD